samples/msal-node-auth-code/index.js

```diff
@@ -4,12 +4,42 @@
  */
 const express = require("express");
 const msal = require('@azure/msal-node');
-const myLocalCache = require("./data/cache");
 const fs = require("fs");
 
 const SERVER_PORT = process.env.PORT || 3000;
 
-// initialize msal public client application
+
+const readFromStorage = () => {
+    return new Promise((resolve, reject) => {
+        fs.readFile("./data/cache.json", (err, data) => {
+            if (err) {
+                reject(err);
+            } else {
+                resolve(data);
+            }
+        });
+    });
+};
+const writeToStorage = (getMergedState) => {
+    return new Promise((resolve, reject) => {
+        readFromStorage().then(oldFile => {
+            const mergedState = getMergedState(oldFile);
+            fs.writeFile("./data/cache.json", mergedState, (err, data) => {
+                if (err) {
+                    reject(err);
+                } else {
+                    resolve();
+                }
+            });
+        });
+    });
+};
+
+const cachePlugin = {
+    readFromStorage: readFromStorage,
+    writeToStorage: writeToStorage
+};
```

Comment:
```suggestion
const cachePlugin = {
    readFromStorage,
    writeToStorage
};
```

Reply:
does this work in the sample without and transpilation?

Reply:
Latest versions of Node support this out-of-the-box.